modules/st2-elastic-field/directive.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+angular.module('main')
+  .directive('st2ElasticField', [
```

Comment:
Another thing with `st2ElasticField` is that I'm starting to wonder, if we're making it a separate directive, where else can we use it? From line 115, it seems like we're only limited by another textarea element and that's it.

At the moment, I don't see another use for this directive and if you also don't have anything in mind, I suggest we move the whole logic into `st2FormTextField` directive.